dali/operators/reader/file_reader_op.cc

```diff
@@ -31,8 +31,10 @@ directories with images in them. To obtain labels `FileReader` sorts directories
 `file_root` in alphabetical order and takes an index in this order as a class label.)code",
       DALI_STRING)
   .AddOptionalArg("file_list",
-      R"code(Path to the file with a list of pairs ``file label``
-(leave empty to traverse the `file_root` directory to obtain files and labels))code",
+      R"code(Path to a text file containing rows of `filename label` pairs, where the filenames are
+relative to `file_root`.
+If left empty, `file_root` is traversed for subdirectories containing files associated with
+the same label. When traversing subdirectories, labels are assigned consecutive numbers.)code",
```

Comment:
Few things here:
1. You need double-back-ticks here
2. How about adding info, that traversing works only for 1-level-deep structure?

Reply:
Done